---
title: "Bisulphite-Sequencing"
author: "cacerv, bubibyte, julianschmocker, tseong"
date: "25 9 2019"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```


Epigenetic mechanisms may lead to different expressions of identical DNA sequences. One of these mechanisms is Cytosin methylation, repressing transcriptional activity.
These DNA alterations can be studied via **Bisulphite-Sequencing**: 

## Technology

Bisulphite treatment of DNA converts cytosine, but not methylated cytosine, to uracil. Subsequent high-throughput sequencing yields high-resolution resolution information about the methylation status of the DNA. [^1]

## Application

A comprehensive epigenome analysis can reveal mechanisms of epigenomic changes in disease and aging. It can also reveal aspects of gene-environment interaction as the epigenome is highly influenced by the environment.

## Statistics

Complete linkeage hierarchical clustering, Pearson correlation [^1]

Regression-based methods, e.g. methylKit, BiSeq, 
Smoothing-based methods, e.g. BSmooth 
Hidden Markov models: HMM-DM and HMM-Fisher [^2]

## References

[^1]: Lister et al, Human DNA methylomes at base resolution show widespread epigenomic differences, Nature, 2009.
[^2]: Yu et al, Comparing five statistical methods of differential methylation identification using bisulfite sequencing data, Stat Appl Genet Mol Biol., 2016
